packages/picasso/src/Autocomplete/Autocomplete.tsx

```diff
@@ -85,8 +85,7 @@ export interface Props
   startAdornment?: ReactNode
   /** ReactNode for labels that will be used as end InputAdornment - */
   endAdornment?: ReactNode
-  /** @deprecated */
-  /** Indicate whether `Autocomplete` is in error state */
+  /** @deprecated Indicate whether `Autocomplete` is in error state */
```

Comment:
shouldn't be `@deprecated` on separate line? 🤷‍♂️ 
```suggestion
  /**
   * @deprecated
   * Indicate whether `Autocomplete` is in the error state
   */
```